packages/client/internals/SlidesOverview.vue

```diff
@@ -157,6 +158,7 @@ watchEffect(() => {
     </div>
   </Transition>
   <button v-if="value" class="fixed text-2xl top-4 right-4 slidev-icon-btn text-gray-400" @click="close">
+    <HiddenText text="Close" />
```

Comment:
I wonder if in this case, would a `title` to the button would be more straightforward?

Reply:
I think the title attribute isn't the best way to make thinks more accessible (even if at the first look, I thought the same). See for example: https://www.a11yproject.com/posts/title-attributes/#:~:text=Short%20answer%3A%20Don't%20use,and%20possibly%20not%20even%20used.

It could be tackled with aria attributes as well, but there are also some older screenreaders and browser *caught* IE 😃 having problems with hidden elements.
The CSS trick is a pretty old but effective one which should work for almost every device and browser.